File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1071.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

x = parseInt(lines.shift()), y = parseInt(lines.shift()), soma=0;

if(x>y){
    temp=x;
    x=y;
    y=temp;
}
if(x%2==0){
    x++;
} else{
    x+=2;
}

for(i=x;i<y;i+=2){
    soma+=i;
}
console.log(soma);

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1071.py
soma=0
x = int(input())
y = int(input())

if(x>y):
    temp=x
    x=y
    y=temp

if(x%2==0):
    x=x+1
else:
    x=x+2

for i in range(x, y, 2):
    soma=soma+i

print(soma)
